src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -131,7 +144,7 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
                                     fs.Write(buffer, 0, nBytesRead);
 
                                     ResetCurrentConsoleLine(vTermMode.IsEnabled);
-                                    Console.Out.Write($"\tRecording trace {GetSize(fs.Length)}");
+                                    Console.Out.Write($"[{stopwatch.Elapsed.Seconds,3}s]\tRecording trace {GetSize(fs.Length)}");
```

Comment:
Traces could record for a long time in the limit (days easily, weeks or months don't seem crazy). I think we should use a longer duration time format if we are going to have one.

Reply:
With this most recent change I used a `TimeSpan` instead.  You can specify timespans as `dd:hh:mm:ss`.